packages/api/src/services/popular_reads.ts

```diff
@@ -88,6 +94,37 @@ export const addPopularRead = async (
   return pageId
 }
 
+const addPopularReads = async (
+  userId: string,
+  ...names: string[]
+): Promise<AddPopularReadResult[]> => {
+  const results: AddPopularReadResult[] = []
+  for (const name of names) {
+    const pageId = await addPopularRead(userId, name)
+    results.push({
+      pageId,
+      name,
+      status: pageId
+        ? ArticleSavingRequestStatus.Succeeded
+        : ArticleSavingRequestStatus.Failed,
+    })
+  }
+  return results
+}
+
+export const addPopularReadsForNewUser = async (
+  userId: string,
+  isIOSUser = false
+): Promise<void> => {
+  await addPopularReads(
+    userId,
+    'omnivore_get_started',
+    'power_read_it_later',
+    'omnivore_organize',
+    isIOSUser ? 'omnivore_ios' : 'omnivore_android'
```

Comment:
I think this is wrong, because the user could be web. What we might want to do is replace isIOSUser with `client` and then have something like:

```
defaultReads = ['omnivore_get_started', 'power_read_it_later', 'omnivore_organize']

var reads = defaultReads

switch(client) {
  case iOS:
     reads += 'omnivore_ios'
}
```

One other thing we could consider is language too, but that might be too complicated for now.

Reply:
This is just a `TODO` because I haven't got the starting guide for iOS user so I just put rubbish here :)